frontend-next-migration/src/shared/ui/CustomForm/ui/CustomForm.tsx

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
> *Caution: Added 'use client' to CustomForm.tsx for pass the build!

You could have used simple if else magic instead of `useState` & `useEffect` to keep CustomForm.tsx server sided. But since whatever CustomForm is connected to is most likely a client component, it wouldnt really matter anyway. Its just needed to pass the build really, which I understand. (if client side component existed and used this dropdown, build would pass even without declaring `use client` in `CustomForm.tsx`)